src/custom_request_schedule_manager.h

```diff
@@ -55,7 +55,8 @@ class CustomRequestScheduleManager : public RequestRateManager {
   /// object
   ///
   static cb::Error Create(
-      const pa::PAParamsPtr& params, const std::shared_ptr<ModelParser>& parser,
+      const PerfAnalyzerParameters& params,
```

Comment:
I'm a fan of passing references over smart pointers, but why was this change needed?

Reply:
For the unit test, I don't remember quite correctly, but the unit test couldn't access the pointer.
I will take a look later to refresh my memory and correctly define what was going wrong.